x-pack/docs/en/security/authorization/privileges.asciidoc

```diff
@@ -47,7 +47,7 @@ patterns. It also includes the authority to grant the privileges necessary to
 manage follower indices and auto-follow patterns. This privilege is necessary 
 only on clusters that contain follower indices. 
 
-`manage_data_frame_transforms`::
+`manage_transform`::
```

Comment:
Nit: "All operations related to managing {transforms}." Should we say {transform}? Plural has changed to singular for the feature.

Reply:
@hendrikmuhs I see your point. However, the user will manage transforms in general, not a single transform. Because of this, I think "All operations related to managing a transform" would be strange in this particular case. If you find the singular solution better, I'll change it though.